outline_vpn/utils.py

```diff
@@ -0,0 +1,21 @@
+
+
```

Comment:
These should not be empty. You might want to try formatting the code using [black](https://github.com/psf/black)